workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/RequiredFieldInjector.cs

```diff
@@ -76,16 +82,16 @@ public void DeInjectAllReadersWriters(MonoBehaviour behaviour)
             }
         }
 
-        public List<uint> GetRequiredReaderComponentIds(Type behaviourType)
+        public List<uint> GetComponentPresenceRequirements(Type behaviourType)
         {
             EnsureLoaded(behaviourType);
-            return componentReaderIdsForBehaviours[behaviourType];
+            return componentReadRequirementsForBehaviours[behaviourType];
```

Comment:
There are some naming inconsistencies with `ComponentPresenceRequirements` and `ComponentReadRequirements`.

Reply:
True, the result of my indecision since I found "presence" a better description but longer so I kind of used it most the time except when the variable names were already very long... Should keep consistency though, will fix